src/node_sea.cc

```diff
@@ -238,6 +278,49 @@ std::optional<SeaConfig> ParseSingleExecutableConfig(
   return result;
 }
 
+std::optional<std::string> GenerateCodeCache(std::string_view main_path,
+                                             std::string_view main_script) {
+  RAIIIsolate raii_isolate;
+  Isolate* isolate = raii_isolate.get();
+
+  HandleScope handle_scope(isolate);
+  Local<Context> context = Context::New(isolate);
+  Context::Scope context_scope(context);
+
+  errors::PrinterTryCatch bootstrapCatch(
+      isolate, errors::PrinterTryCatch::kPrintSourceLine);
+
+  Local<String> filename;
+  if (!String::NewFromUtf8(isolate, main_path.data()).ToLocal(&filename)) {
+    return {};
```

Comment:
Shouldn't this be `std::nullopt`?

Reply:
Both `std::nullopt` and `{}` are equivalent here but since `std::nullopt` is clearer, I've done that